src/workflows/continue.js

```diff
@@ -0,0 +1,34 @@
+import * as run from "./steps";
+
+export default [
+	run.verifyConflictResolution,
+	run.gitStageFiles,
+	run.gitRebaseContinue,
+	run.getFeatureBranch,
+	run.setPromote,
+	// default tag-release flow
```

Comment:
As we have talked about previous it could be helpful to pull out a common group of tasks like "`defaultFlow`" and make that into its own array to be pulled in. I'll make this into an issue that we can track